### PR TITLE
Feature/21 logs management

### LINKED ISSUE #21: Feature: Logs management
Similar to http://localhost:1337/admin/logs

We should be able to change the logs of other applications

### PR DESCRIPTION
**Pull request by nassimerrahoui**
Todo list :

- [x] Unit tests for routes component
- [x] Unit tests for loggers component

Fix #21

### CODE REVIEW

File: src/main/webapp/app/admin/logs/logs.component.ts [hunk 1]
```diff
@@ -2,8 +2,13 @@ import { Component, Inject } from 'vue-property-decorator';
 import { mixins } from 'vue-class-component';
 import Vue2Filters from 'vue2-filters';
 import LogsService from './logs.service';
+import RoutesSelectorVue from '@/shared/routes/routes-selector.vue';
```
Comment (pascalgrimaud) [inline src/main/webapp/app/admin/logs/logs.component.ts:5]: why these changes for this file ?

File: src/main/webapp/app/admin/logs/logs.vue [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <template>
-    <div class="table-responsive">
-        <h2 id="logs-page-heading">Logs</h2>
+    <div>
```
Comment (pascalgrimaud) [inline src/main/webapp/app/admin/logs/logs.vue:2]: changes not related to this PR. If needed, it should be done in a separate PR and maybe to Vue.js blueprint too

Comment (nassimerrahoui) [reply src/main/webapp/app/admin/logs/logs.vue:2]: Yes, we need to seperate new loggers component for instances to get less conflict when we upgrade the app. In my next commit I will reset this changes.

File: src/main/webapp/app/admin/logs/logs.component.ts (hunk 1 above)
Comment (nassimerrahoui) [reply src/main/webapp/app/admin/logs/logs.component.ts:5]: I will reset this change. It is not necessary here, because I created a new component for instance's logs.

**Comment by pascalgrimaud** [comment]
plz, add `fix https://github.com/jhipster/jhipster-control-center/issues/21` in the description of the PR

**Comment by pascalgrimaud** [comment]
As the PR related to sidebar has been merged, can you resolve the conflict plz ?
The new item loggers should be in sidebar now, instead of navbar

**Comment by nassimerrahoui** [comment]
> As the PR related to sidebar has been merged, can you resolve the conflict plz ?
> The new item loggers should be in sidebar now, instead of navbar

Okay, I will resolve it in my next commit with unit tests for loggers component

**Comment by nassimerrahoui** [comment]
My bad for BSidebar, it works with import from bootstrap-vue, it was a problem with my IDE. Don't take into account my last message in commit named : "Resolve conflict after merge of pr 56"

**Comment by nassimerrahoui** [comment]
Closing this PR to clean commits.
